checklist.md

```diff
@@ -0,0 +1,105 @@
+# Сheck Your Code Against the Following Points
+
+## Code Efficiency
+
+Don't compare a variable to `0` if there is no need:
+
+Good example:
+
+```python
+if number_of_students:
+    pass
+```
+
+Bad example:
+
+```python
+if number_of_students != 0:
+    pass
+```
+
+## Code Style
+
+1. Use one style of quotes in your code. Double quotes are preferable.
+2. Use `( )` while importing multiple classes/modules:
+
+Good example:
+
+```python
+from app.main import (
+    KnightWithArmour,
+    KnightWithoutArmour,
+    SuperHeroKnightWithGun
+)
+```
+
+Bad example:
+
+```python
+from app.main import KnightWithArmour, \
+    KnightWithoutArmour, \
+    SuperHeroKnightWithGun
+```
+
+3. Use descriptive error messages:
+
+Good example:
+
+```python
+if error:
+    raise CustomError("We have an error on the line 16.")
```

Comment:
may be we can cower it with test?

Reply:
good idea

Reply:
but if I'm not mistaken, students can write different error messages

Reply:
check, the message exists or not